apps/website/src/examples/Tabs.overflow.tsx

```diff
@@ -0,0 +1,95 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Tabs, Tab, DropdownMenu, MenuItem, IconButton } from '@itwin/itwinui-react';
+import SvgMoreSmall from '@itwin/itwinui-icons-react/cjs/icons/MoreSmall';
```

Comment:
Can you share a screenshot of what it looks like with `SvgMore`, not the small variant?  I think it might look more proportional.

Reply:
I did the Breadcrumbs version with SvgMore and I was asked to make it the small version.

Here is what it looks like with the big version, vertically centered:
![image](https://user-images.githubusercontent.com/37936169/217630415-a61899f5-d191-4346-b076-c7ca64e434d9.png)

Reply:
Here's the Breadcrumbs version I did before: 
![image](https://user-images.githubusercontent.com/37936169/217631314-b3779c04-d07c-41e7-b3de-28577ddefc65.png)
 I'm noticing now that it's not exactly lined up with the text, but I tried.

Reply:
No, the ellipsis are clickable. See https://github.com/iTwin/iTwinUI-react/pull/814
![image](https://user-images.githubusercontent.com/37936169/217842676-23781daa-f5c4-41a7-8d8b-d2f0e4fad08f.png)
At one point the '...' icon was blue (so that it looked like a button), so I'm not sure why it's black now.

Reply:
But, look, I'll change both to be SvgMore and centered. But dang, it's frustrating to get the exact opposite feedback on something so similar, haha

Reply:
Oh wow, I was unaware.  The [breadcrumb HTML demos](https://itwin.github.io/iTwinUI/css/breadcrumbs.html) are using the ellipsis text, not an SVG.  If we're using an SVG in storybook, we should probably have a demo using the SVG in our HTML demos as well.

Reply:
Update on the blue/not blue situation: It's blue in react v1, and black in react v2. Did all the button colors change or something?

Reply:
Updated